### PR TITLE
chore: use actor cids from the manifest

### PR DESCRIPTION
**Pull request by LesnyRumcajs**
## Summary of changes

Changes introduced in this pull request:

- use CIDs from our self-generated manifest and not from Go-parsed code.

## Reference issue to close (if applicable)

Closes https://github.com/ChainSafe/forest/issues/4347

## Other information and links

~~Can be reviewed now, but blocked on https://github.com/ChainSafe/fil-actor-states/pull/320 to adjust the `fil-actor-states` version.~~

~~Also, tests blocked by https://github.com/ChainSafe/forest-iac/pull/460~~

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by hanabi1224 — APPROVED** [review]

File: src/shim/actors/mod.rs [hunk 1]
```diff
@@ -8,3 +8,8 @@ pub mod multisig;
 pub mod verifreg;
 
 pub use common::*;
+
+pub mod state_load;
+pub use state_load::*;
+mod version;
+pub use version::*;
```
Comment (hanabi1224) [inline src/shim/actors/mod.rs:15]: How about adding `prelude` and `use  crate::shim::actors::prelude::*` in other places 
```rust
pub mod prelude {
    pub use super::state_load::*;
}
```

File: Cargo.toml [hunk 2]
```diff
@@ -47,18 +47,18 @@ digest = "0.10"
...
-fil_actor_init_state = { version = "15.2" }
-fil_actor_interface = { version = "15.2" }
-fil_actor_market_state = { version = "15.2" }
-fil_actor_miner_state = { version = "15.2" }
-fil_actor_power_state = { version = "15.2" }
-fil_actor_reward_state = { version = "15.2" }
-fil_actor_system_state = { version = "15.2" }
-fil_actor_verifreg_state = { version = "15.2" }
-fil_actors_shared = { version = "15.2", features = ["json"] }
+fil_actor_account_state = { git = "https://github.com/ChainSafe/fil-actor-states", branch = "no-cids-in-here" }
```
Comment (hanabi1224) [inline Cargo.toml:50]: use published versions

Comment (hanabi1224) [inline Cargo.toml:50]: use published versions

Comment (LesnyRumcajs) [reply Cargo.toml:50]: As mentioned in the description, this is blocked by https://github.com/ChainSafe/fil-actor-states/pull/320

File: src/shim/actors/mod.rs (hunk 1 above)
Comment (LesnyRumcajs) [reply src/shim/actors/mod.rs:15]: I'm not convinced that adding a prelude with a single wildcard import is useful. I already used `use crate::shim::actors::state_load::*` in places where lots of different actor state loading was required.

We might want to add a bigger prelude later, depending on the usage, with changing the visibility of other modules here, but I wouldn't do it now.

**Review by ruseinov — APPROVED** [review]

**Pull request merged**